packages/aws-cdk/lib/api/util/sdk.ts

```diff
@@ -256,3 +276,16 @@ async function getCLICompatibleDefaultRegion(profile: string | undefined): Promi
 
     return region;
 }
+
+/**
+ * Find and return the configured HTTPS proxy address
+ */
+function httpsProxyAddress(): string | undefined {
+    if (process.env.http_proxy) {
+        return process.env.http_proxy;
+    }
+    if (process.env.https_proxy) {
```

Comment:
I think https should have precedence

Reply:
Yes no, the http_ one should never have been there.